packages/openneuro-app/src/scripts/users/__tests__/datasest-card.spec.tsx

```diff
@@ -183,19 +24,115 @@ const mockDataset = {
     issues: [{ severity: "low" }],
     created: "2025-01-01T00:00:00Z",
     description: {
+      Name: "Test Dataset Description",
       Authors: ["John Doe"],
       SeniorAuthor: "Dr. Smith",
       DatasetType: "fMRI",
     },
+    summary: {
+      modalities: ["fMRI"],
+      secondaryModalities: [],
+      sessions: 1,
+      subjects: 1,
+      subjectMetadata: [],
+      tasks: ["rest"],
+      size: 1024 ** 3,
+      totalFiles: 10,
+      dataProcessed: true,
+      pet: null,
+    },
+    validation: {
+      errors: [],
+      warnings: [],
+    },
   },
+  uploader: {
+    id: "uploaderId123",
+    name: "Uploader Name",
+    orcid: "1234-5678-9012-3456",
+  },
+  permissions: {
+    id: "somePermId",
+    userPermissions: [
+      {
+        userId: "someUserId",
+        level: "admin",
+        access: "admin",
+        user: {
+          id: "someUser",
+          name: "Some User",
+          email: "some@user.com",
+          provider: "github",
+        },
+      },
+    ],
+  },
+  metadata: { ages: [20] },
+  snapshots: [
+    {
+      id: "ds000001:1.0.0",
+      created: "2025-01-01T00:00:00Z",
+      tag: "1.0.0",
+    },
+  ],
 }
 
 describe("DatasetCard", () => {
   it("should render dataset information correctly", () => {
     render(<DatasetCard dataset={mockDataset} hasEdit={false} />)
-
     expect(screen.getByText("Test Dataset")).toBeInTheDocument()
     expect(screen.getByText("ds000001")).toBeInTheDocument()
-    expect(screen.getByText("1.00 GB")).toBeInTheDocument()
+
+    expect(screen.getByText((content, element) => {
+      const isDateSpan = element?.tagName.toLowerCase() === "span" &&
+        element.textContent?.includes("Added:")
+      if (!isDateSpan) return false
+      const expectedDatePart = "12/31/2024"
+      const expectedAgoPart = "4 months ago"
```

Comment:
Age is misspelled and this test is dependent on the current date, so this test will fail periodically. Skip matching on this string in the test is easiest, I don't think this is a very useful thing to check if the date itself is rendered.